tests/test_models.py

```diff
@@ -32,3 +39,13 @@ def test_openai_model(openai_mock_client):
                 model = OpenAIModel(TEST_MODEL_ARGUMENTS, [])
         model.client = openai_mock_client
         model.query(TEST_HISTORY)
+
+
+@patch("together.Complete.create", return_value=together_return_value())
+def test_together_model(mock):
```

Comment:
Now this unit test is a little different from OpenAI's unit tests. But I'm looking for a test format that can be reused even when adding other models or new models. I'm aiming to standardize this in issue #185.